English Translations [will work on A LOT MORE later]

byronbytes opened the pull request:
currently am busy right now

byronbytes committed:
Create en.yml

byronbytes committed:
Added all Error labels (i think)

I am busy today

byronbytes committed:
English Fixes in readme

There are more, but I will get to them tomorrow.

Potato1682 reviewed (changes requested):
You need to change the entire document.

byronbytes commented on docs/README-en.md:
```diff
@@ -131,17 +131,17 @@ In the description of **hack**, the meaning of hacking (cracking) of the server
 Please note that this project is released with a [Contributor Code of Conduct](CODE-OF-CONDUCT-en.md). By participating in this project you agree to abide by its terms.
 
-### Markdown formatting (This area is not so important)
+### Markdown Formatting (Not so important)
```
I am confused by this statement

byronbytes committed:
More English Translations

There might be some broken English on my end, because I don't know context in some places.

Potato1682 commented:
This is duplicated.
